decentraland/unity-renderer
# Fix: hiccups when opening the explore menu

## What does this PR change?
Fix #2061 

- Replaced `GameObject.SetActive` by `Canvas.enabled` logic for subSections
- Replaced instant loading of cards with sequential loading (throttling) - for initial loading and also for LoadMore button logic
- Refill Pools when they get emptied (also with throttling)
- Removed SpriteMeshGeneration on sprite creation
- Moved logic of hiding loading bar from Controller inside View

## How to test the changes?

1. Go to: https://play.decentraland.zone/?renderer-branch=bug/hiccups-when-opening-the-explore-menu
2. Open ExploreV2 events section
3. Switch between Highlights->Places->Events
4. Press LoadMore button in Places and Events
5. Observe improved performance

Devs-only: you can compare all 3 main improved behaviours with profiling current Dev and this branch. Main behaviours
- Initial loading of cards (for Highlights and Places are easier to check, because Events almost loaded immidiatly)
- Switching between subSections
- Loading More button behaviour

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
